langchain/tools/toolkit.py

```diff
@@ -0,0 +1,17 @@
+"""Toolkits form collections of tools that can be executed by an LLM."""
+
+
+from abc import abstractmethod
+from typing import List
+
+from pydantic import BaseModel
+
+from langchain.tools.tool import Tool
+
+
+class Toolkit(BaseModel):
```

Comment:
Toolkit Definition here. Plus tool definitions in this folder

Reply:
Would call this `BaseToolkit`